storage/storage.rs

```diff
@@ -206,18 +215,50 @@ impl<D> MVCCStorage<D> {
         D: DurabilityService,
     {
         use StorageRecoverError::DurabilityServiceRead;
-        let records = self
-            .durability_service
-            .iter_type_from_start::<CommitRecord>()
+        let records =
+            IsolationManager::iterate_commit_status_from_disk(&self.durability_service, SequenceNumber::MIN)
```

Comment:
This method doesn't feel like it belongs to `IsolationManager`, since it's nothing to do with isolation. However, this also doesn't really fit in `storage` -- this is hinting maybe we're missing an abstraction.

I think if we think about a networked durability service, maybe it's time to introduce a `DurabilityManager` in `//storage`, which holds this method and maybe some other ones? If you make it, what falls into it naturally?